app/controllers/planning_applications/assessment/consideration_guidances_controller.rb

```diff
@@ -76,6 +77,10 @@ def set_considerations
         @considerations = @consideration_set.considerations.select(&:persisted?)
       end
 
+      def set_review
+        @review = @consideration_set.current_review
+      end
```

Comment:
I'm setting the review here but the form submission goes to the original considerations controller - something to think about if we switch to the pre-app design for considerations on full applications.